static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -45,7 +45,7 @@ const fillAdvertSlots = (
     start: () => void,
     stop: () => void
 ): Promise<void> => {
-    if (commercialFeatures.dfpAdvertising) {
+    if (commercialFeatures.dfpAdvertising || commercialFeatures.adFree) {
```

Comment:
Yes. `adFree` suppresses `dfpAdvertising` by default because that is used to activate many other features.